src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  const gender = sex === 'm'
+    ? <p className="Person__partner">{`${partnerName} is my wife`}</p>
+    : <p className="Person__partner">{`${partnerName} is my husband`}</p>;
+
+  return (
+    <div className="App">
+      <section className="Person">
+        <h2 className="Person__name">{`My name is ${name}`}</h2>
+        {age && (
+          <p className="Person__age">{`I am ${age}`}</p>
+        )}
+
+        {
+        isMarried === true
```

Comment:
```suggestion
        {isMarried
```

sorry, better not to break the line, but the comparison is redundant here,